src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -90,7 +90,7 @@ public Socket(AddressFamily addressFamily, SocketType socketType, ProtocolType p
             s_loggingEnabled = SocketsEventSource.Log.IsEnabled();
             if (s_loggingEnabled)
             {
-                NetEventSource.Enter(NetEventSource.ComponentType.Socket, this, "Socket", addressFamily);
+                NetEventSource.Enter(NetEventSource.ComponentType.Socket, this, nameof(Socket), addressFamily);
```

Comment:
Yes, there's a whole issue about overhauling how this logging is done.  That would hopefully be a piece of it.